tools/kmod/sof_insert.sh

```diff
@@ -4,19 +4,29 @@
 
 insert_module() {
 
-    local MODULE="$1"
+    local MODULE="$1"; shift
```

Comment:
shift? is this required?

Reply:
Yes to remove the module name from "$@" below